src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const delRow = document.querySelector('.remove-row');
+const addCol = document.querySelector('.append-column');
```

Comment:
it's a good practice when you add suffix `El` or `Ref` to the variables with node element
Example: `const tableRef = document.querySelector('.field');`